src/app/modules/form/admin-form/__tests__/admin-form.controller.spec.ts

```diff
@@ -488,50 +493,13 @@ describe('admin-form.controller', () => {
       expect(MockUserService.getPopulatedUserById).toHaveBeenCalledWith(
         MOCK_USER_ID,
       )
-      expect(MockFormService.retrieveFullFormById).toHaveBeenCalledWith(
-        MOCK_FORM_ID.toHexString(),
-      )
-      expect(
-        MockSubmissionService.getFormSubmissionsCount,
-      ).not.toHaveBeenCalled()
-      expect(mockRes.status).toHaveBeenCalledWith(410)
-      expect(mockRes.json).toHaveBeenCalledWith({
-        message: expectedErrorString,
-      })
-    })
-
-    it('should return 410 when FormDeletedError is returned when checking form availability', async () => {
```

Comment:
why is this test deleted?

Reply:
the new `AuthService#getFormAfterPermissionChecks` already tests for this. Since this controller level test is only mocking the result of the auth service, then no need for this test